doc/man3/PBMAC1_get1_pbkdf2_param.pod

```diff
@@ -0,0 +1,46 @@
+=pod
+
+=head1 NAME
+
+PBMAC1_get1_pbkdf2_param - Function to manipulate a PBMAC1
+MAC structure
+
+=head1 SYNOPSIS
+
+ #include <openssl/x509.h>
+
+ PBKDF2PARAM *PBMAC1_get1_pbkdf2_param(const X509_ALGOR *macalg);
+
+=head1 DESCRIPTION
+
+PBMAC1_get1_pbkdf2_param() retrieves B<PBKDF2PARAM> struct from I<X509_ALGOR>
+structure.
+
+=head1 RETURN VALUES
+
+PBMAC1_get1_pbkdf2_param() return NULL in case when PBMAC1 uses an
+algorithm different from B<PBKDF2> or incorrect parameters and a pointer to
+B<PBKDF2PARAM> struct otherwise.
```

Comment:
PBMAC1_get1_pbkdf2_param() return**s** NULL when PBMAC1 uses an
algorithm ~different~ **apart** from B<PBKDF2> or **when passed** incorrect parameters and a pointer to
B<PBKDF2PARAM> struct**ure** otherwise.

Reply:
Done, thanks